llvm/lib/CodeGen/GlobalISel/LegalizerHelper.cpp

```diff
@@ -3690,7 +3690,7 @@ LegalizerHelper::LegalizeResult LegalizerHelper::lowerLoad(GAnyLoad &LoadMI) {
 
   if (!isPowerOf2_32(MemSizeInBits)) {
```

Comment:
Doesn't this line need to be be updated if MemSizeInBits is a scalable size?

Reply:
Indeed, and I'm still pondering how to do it.
Don't have a super clear idea about how to do it. Partly because all of out test cases are captured in the `custom` branch so they will enter this `lower` branch (to be honest, I can just revert all my current changes in this file and support it later when there's an actual need); also partly because I don't quite understand why they require this load splitting into power of 2 sized loads.

Reply:
Ok let's drop the changes if they aren't tested.